app/assets/javascripts/components/restaurantList.js.jsx

```diff
@@ -55,6 +73,14 @@ class RestaurantList extends React.Component {
           <span>Filter Results:</span>
           <input type="text" placeholder="Filter by name"
                    onChange={this.handleNameFilterChanged.bind(this)} className="filter"/>
+          <select onChange={this.handleCuisineChange.bind(this)} className="filter">
+            <option value={this.allCuisines}>{this.allCuisines}</option>
```

Comment:
you'd probably want a different value for this option instead of re-using the placeholder text